packages/webdriverio/src/commands/browser/uploadFile.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * Uploads a file to the selenium server by using the [`file`](/api/protocol/file.html) command. Note that
+ * this command might not be supported as it is an undocumented Selenium command.
```

Comment:
Can we add a code example with an actual page like https://the-internet.herokuapp.com/upload ?